psga/common/contamination_removal.nf

```diff
@@ -24,7 +24,7 @@ process contamination_removal_ont {
 
   ref_genome_fasta=!{ref_genome_fasta}
   file_1=!{file_1}
-  sample_id=$( echo ${file_1} | cut -d '_' -f1 )
```

Comment:
This is a minor code fix. It should have been `.` not a `_`, otherwise the sample_id becomes the whole file name.